build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenLocal()
```

Comment:
Why do we need this for the buildscript?

Reply:
removed.